services/src/test/scala/cromwell/services/womtool/impl/WomtoolServiceInCromwellActorSpec.scala

```diff
@@ -153,20 +153,20 @@ class WomtoolServiceInCromwellActorSpec extends ServicesSpec("Womtool") {
       )
     }
 
-    "return invalid for a valid workflow with the wrong inputs" in {
+    "return inputErrors for a valid workflow with the wrong inputs" in {
 
       val wsfc = wsfcConjurer(workflowSource = Option(TestData.wdlValid), inputsJson = TestData.bogusInputs)
 
       check(DescribeRequest(wsfc), DescribeSuccess(
-        description = WorkflowDescription(valid = false, errors = List("Required workflow input 'wf_hello.hello.addressee' not specified"))))
+        description = WorkflowDescription(valid = true, inputErrors = List("Required workflow input 'wf_hello.hello.addressee' not specified"), name = "wf_hello", inputs = List(InputDescription("wf_hello.hello.addressee", WomStringType, "String", false, None)), submittedDescriptorType = Map("descriptorType" -> "WDL", "descriptorTypeVersion" -> "1.0"))))
```

Comment:
Oh interesting. I'm... unsure... whether I'm cool with `valid = true` coexisting with `inputErrors`...

Reply:
Hmm, this sounds like a good suggestion. I'll give it a try.